guides/gitconfig.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Managing Git Configuration
+description: Learn how to manage your Git configuration in Coder.
```

Comment:
I Think we can opt for brevity here, something along the lines of:

```
Learn how to configure Git with Coder
```

Though I'd ultimately leave that decision to @khorne3